pkg/sfu/bwe/null_bwe.go

```diff
@@ -62,6 +62,10 @@ func (n *NullBWE) ProbeClusterStarting(_pci ccutils.ProbeClusterInfo) {}
 
 func (n *NullBWE) ProbeClusterDone(_pci ccutils.ProbeClusterInfo) {}
 
+func (n *NullBWE) ProbeClusterIsGoalReached() bool {
+	return false
+}
```

Comment:
Haven't done this for TWCC case yet. Need to do more work around congestion state machine oscillating too much in TWCC case when using loss only congestion detection. Need to dig into it. Will implement this when I look at that.